lib/widgets/message_list.dart

```diff
@@ -322,6 +322,8 @@ class MessageListAppBarTitle extends StatelessWidget {
   Widget build(BuildContext context) {
     final zulipLocalizations = ZulipLocalizations.of(context);
 
+    // TODO(i18n): provide tranlations just as 'zulipLocalizations.unknownUserName'
+    // for 'unknown channel'
```

Comment:
nit: `tranlations` -> `translations`

Reply:
It would be better to move this comment to right before any of the lines containing `(unknown channel)`, so it is easier to spot where the translation is applicable.

Reply:
Okay :+1: , just noticed i didn't spell that right